playwright/e2e/validateJsuiVersion.ts

```diff
@@ -0,0 +1,24 @@
+// This test is run in CI to validate that the JSUI version is the one expected.
+import {test, expect} from '@playwright/test';
+import {pageURL} from '../utils/utils';
+
+const expectedJsuiVersion = process.env.JSUI_VERSION;
+const timeout = 180_000;
+
+if (expectedJsuiVersion) {
+    console.log(`Expected JSUI version: ${expectedJsuiVersion}`);
+
+    test('validate JSUI version', async ({page}) => {
+        test.setTimeout(timeout);
+        await expect(async () => {
+            await page.goto(pageURL());
+            const coveoVersion = await page.evaluate(() => (window as any).Coveo.version);
+            // Example value: {lib: '2.10120.0', product: '2.10120.0', supportedApiVersion: 2}
+            expect(coveoVersion.lib).toBe(expectedJsuiVersion);
+            expect(coveoVersion.product).toBe(expectedJsuiVersion);
```

Comment:
Looks like these tests are failing/flaky ?

Reply:
The errors here are from the test I did where ran with JSUI_VERSION `2.10120.1` (doesn't exist) to test the failing scenario.

The errors a but further down are actual flakiness. 
@lvu285 maybe you could look at that in another PR?

Reply:
for some reason, on those days that you ran the test, the facet ShowMore has issue _ is flaky from backend. It should be back to normal